ivy_tests/test_ivy/test_frontends/test_torch/test_loss_functions.py

```diff
@@ -563,6 +563,79 @@ def test_torch_nll_loss(
     )
 
 
+# gaussian_nll_loss
+@handle_frontend_test(
+    fn_tree="torch.nn.functional.gaussian_nll_loss",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_value=0.01,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=1,
+        min_dim_size=1,
+        max_dim_size=1,
+    ),
+    dtype_and_target=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("integer"),
+        min_value=0.0,
+        max_value=1.0,
+        allow_inf=False,
+        min_num_dims=1,
+        max_num_dims=1,
```

Comment:
Also, right now you're forcing `input`, `target` and `var` arrays of size 1 (just one element). This is not the only case you should be testing for. You need to test for any shape.

Reply:
I have set the max_dim_size = 5. I believe this should be representative for higher dimensional arrays.